packages/graphql/tests/tck/issues/1751.test.ts

```diff
@@ -84,10 +84,18 @@ describe("https://github.com/neo4j/graphql/issues/1751", () => {
             WHERE this.title = $param0
             WITH this
             OPTIONAL MATCH (this)-[this_admins0_relationship:HAS_ADMINISTRATOR]->(this_admins0:Admin)
-            WHERE apoc.cypher.runFirstColumnSingle(\\" MATCH (this_admins0)<-[aggr_edge:HAS_ADMINISTRATOR]-(aggr_node:Organization)
-            RETURN count(aggr_node) = $aggr_count
-            \\", { this_admins0: this_admins0, aggr_count: $aggr_count })
+            CALL {
+                WITH this_admins0
+                MATCH (this_deleteOrganizations_args_delete_admins0_where_Adminthis2:\`Organization\`)-[this_deleteOrganizations_args_delete_admins0_where_Adminthis1:HAS_ADMINISTRATOR]->(this_admins0)
+                RETURN count(this_deleteOrganizations_args_delete_admins0_where_Adminthis2) = $this_deleteOrganizations_args_delete_admins0_where_Adminparam0 AS this_deleteOrganizations_args_delete_admins0_where_Adminvar0
+            }
+            WITH *, CASE this_deleteOrganizations_args_delete_admins0_where_Adminvar0 = true
+                WHEN true THEN [ this_admins0_relationship, this_admins0 ]
+                ELSE [ NULL, NULL ]
```

Comment:
It is, in this case, it could be removed by adding `Cypher.Variable` to the `Cypher.Predicate` type, is just not worth in my opinion for the scope of this PR.